### ISSUE TITLE
2020 release wave 1 - An item with the same key has already been added

### ISSUE THREAD

**Issue by chironh** [Innofactor/PluginRegistration#63]
After installing the 2020 release wave 1, the plugin registration tool throws multiple errors when:

- opening "Assembly View"
- registering new image in "Assembly View"

I guess most of those exceptions are related to each other. I was thinking about the Dynamics update having multiple steps with the same GUID, but I wasn't able to identify the problem. 

I can simulate this problem on multiple organizations/tenants.
The MS Plugin Registration Tool doesn't seem to be affected.

```
Unhandled Exception: System.SystemException

An item with the same key has already been added.

   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at Xrm.Sdk.PluginRegistration.Controls.CrmTreeControl.AddNodes(TreeNodeCollection parentNodeList, List`1 nodes, Boolean checkNodes)
   at Xrm.Sdk.PluginRegistration.Controls.CrmTreeControl.AddNodes(TreeNodeCollection parentNodeList, List`1 nodes, Boolean checkNodes)
   at Xrm.Sdk.PluginRegistration.Controls.CrmTreeControl.AddNodes(TreeNodeCollection parentNodeList, List`1 nodes, Boolean checkNodes)
   at Xrm.Sdk.PluginRegistration.Controls.CrmTreeControl.LoadNodes(List`1 rootNodes)
   at Xrm.Sdk.PluginRegistration.MainControl.LoadNodes(CrmViewType view)
```

**Comment by rappen** [comment]
Yep. I get that too.

**Comment by DarylOrwin** [comment]
same

**Comment by imranakram** [comment]
Thanks for letting us know. Latest version should fix this issue v`3.2020.4.14`
